### PR TITLE
feat(converter-nh-accuvote): add `correct-definition` command

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview

Add another command to `converter-nh-accuvote`: `correct-definition`. This command takes pairs of XML and PDF and uses the bubble positions from the PDF to fix the `OX`/`OY` coordinates in the AccuVote definition in the same way our regular conversion does. It outputs the corrected AccuVote definition and a proofing PDF to manually verify the coordinates.

## Demo Video or Screenshot

```
❯ ./bin/correct-definition -h
Usage: correct-definition --config <config-path>

Corrects an AccuVote definition file based on the PDF.

Config File Format:

{
  "cards": [
    {
      // Required: path to the AccuVote definition file
      "definitionPath": "definition.xml",

      // Required: path to the ballot PDF file
      "pdfPath": "ballot.pdf",

      // Required: output directory
      "outputDir": "output"

      // Optional: page numbers (1-based, defaults to 1 & 2,
      // useful for PDFs containing multiple ballot cards)
      "frontPage": 3,
      "backPage": 4,
    }
  ]
}

All paths are relative to the config file or absolute.
```

| PROOF Front | PROOF Back |
|-|-|
| image | image |

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/converter-nh-accuvote/README.md [hunk 1]
```diff
@@ -7,33 +7,36 @@ VotingWorks election definition.
...
-  definition,
-});
-
-if (convertResult.isErr()) {
-  console.error(`error: ${JSON.stringify(convertResult.err())}`);
-} else {
-  console.log('Converted election result:', convertResult.ok().election);
+## CLI Usage
+
+### Correct an AccuVote XML definition
```
Comment (jonahkagan) [inline libs/converter-nh-accuvote/README.md:12]: Do you foresee this always being a separate step? Or might we integrate it into the main command?

File: libs/converter-nh-accuvote/src/convert/correct_definition.ts [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+      continue;
+    }
+
+    pdfReadersByPath.set(
+      pdfPath,
+      new PdfReader(await readFile(pdfPath), { scale: 200 / PDF_PPI })
+    );
+  }
+
+  const results = await Promise.all(
```
Comment (jonahkagan) [inline libs/converter-nh-accuvote/src/convert/correct_definition.ts:182]: Any risk of memory overload from loading all these PDFs concurrently?

File: libs/converter-nh-accuvote/src/convert/correct_definition.ts [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+        side: 'front',
+      })
+    ),
+    bubbles
+  );
+}
+
+/**
+ * Corrects the coordinates of the candidates in the AvsInterface object based on
+ * the PDF file contents.
```
Comment (jonahkagan) [inline libs/converter-nh-accuvote/src/convert/correct_definition.ts:296]: Might be nice to include a short summary of how the correction works

**Review by jonahkagan — APPROVED** [review]

**Comment by eventualbuddha** [comment]
Replaced with https://github.com/votingworks/vxsuite/pull/5304

**Pull request closed without merging**